src/addFunctionWithoutDefaultValue.kt

```diff
@@ -0,0 +1,6 @@
+// "Add 'abstract fun foo(x: String = "")' to 'I'" "true"
+interface I
+
+class Foo : I {
+    <caret>override fun foo(x: String = "") {}
+}
```

Comment:
Override declarations are prohibited to have default parameters. I'd expect the refactoring to generate default values in the base class/interface because. Like this:
```suggestion
interface I {
    fun foo(x: String = "")
}

class Foo : I {
    override fun foo(x: String) {}
}
```